packages/app/src/ErrorFallback.tsx

```diff
@@ -10,7 +10,27 @@ interface Props extends FallbackProps {
 function ErrorFallback(props: Props) {
   const { className = '', error, resetErrorBoundary } = props;
 
-  if (error.cause || error.cause instanceof Error) {
+  if (error.message === CANCELLED_ERROR_MSG) {
```

Comment:
Moving this up because I now use a new utility function called `processAxioError` to throw "Request cancelled" errors, and this utility adds the original error as `cause` when rethrowing. (And I don't need to render an accordion for "Request cancelled" errors, since the cause error message is simply "cancelled".)